InfiniteLoopers-App/src/main/java/com/isa/account/RegistrationForm.java

```diff
@@ -0,0 +1,47 @@
+package com.isa.account;
+
+import java.util.List;
+
+public class RegistrationForm {
+    private UserInputReader userInputReader;
+    private UserManager userManager;
```

Comment:
Mozna to usunac.

Reply:
Dlaczego? Jeżeli usunę, będzie błąd kompilacji